### DISCUSSION TITLE
Why JSON for configs?

### DISCUSSION THREAD

**Discussion by Koncord** [storm-devs/storm-engine#26] [Ideas]
JSON isn't designed for the configurations, by standard, it even does not support comments. In my opinion, would be better to stick with INI or move to the standardized INI - [TOML](https://en.wikipedia.org/wiki/TOML).

**Comment by np-c0mp1ete** [comment]
@Koncord We are definitely open to consider other formats. I deleted the JSON mentioning from the readme and added a link to this discussion instead.
Current config format is far from adhering to any standard. That stops us from throwing away all the custom config parsing code and using some third-party library instead.

**Comment by xakod** [comment]
What about xml? Xml is faster than json/yaml and provides more opportunities than ini.

**Reply by Hammie** [reply]
XML is a lot harder read/write (for humans). 
And json/toml parsers can also be a lot faster then XML parsers, though I do not think it matters that much for our use-case.

**Reply by freeman53** [reply]
You can use yml format

**Reply by Hammie** [reply]
I think toml has a syntax very similar to yml
